app/controllers/ncr/work_orders_controller.rb

```diff
@@ -26,7 +29,7 @@ def update
 
       super
 
-      if @model_changing && !@model_instance.emergency  # skip approvals if emergency
+      if @model_changing && !@model_instance.emergency # skip approvals if emergency
```

Comment:
I think `@model_instance.not_emergency?` would prevent the need for a comment here and would be clearer to read since it's easier to miss the `!`